lib/matplotlib/patches.py

```diff
@@ -1449,6 +1449,43 @@ def get_center(self):
 
     center = property(get_center, set_center)
 
+    def set_width(self, width):
+        '''
+        Set the width of the ellipse
```

Comment:
```suggestion
        Set the width of the ellipse.
```

Please finish the sentences with a period (also in the other methods). I know, there are some missing in the current docs, but we try to move towards a more consistent doc style.